src/thunderbots/software/ai/hl/stp/play/kickoff_friendly_play.cpp

```diff
@@ -0,0 +1,151 @@
+#include "ai/hl/stp/play/kickoff_friendly_play.h"
+
+#include "ai/hl/stp/play/play_factory.h"
+#include "ai/hl/stp/tactic/chip_tactic.h"
+#include "ai/hl/stp/tactic/goalie_tactic.h"
+#include "ai/hl/stp/tactic/move_tactic.h"
+#include "shared/constants.h"
+
+const std::string KickoffFriendlyPlay::name = "KickoffFriendly Play";
+
+std::string KickoffFriendlyPlay::getName() const
+{
+    return KickoffFriendlyPlay::name;
+}
+
+bool KickoffFriendlyPlay::isApplicable(const World &world) const
+{
+    return (world.gameState().isReadyState() | world.gameState().isSetupState()) &&
```

Comment:
bitwise OR???

Reply:
fixed to `||` , forgot to to type the other `|`